ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -722,45 +715,42 @@ public class BottomSheetController: UIViewController {
         return requiredConstraints + [breakableConstraint]
     }
 
-    private lazy var preferredExpandedContentHeightConstraint: NSLayoutConstraint = {
-        let constraint = expandedContentView.heightAnchor.constraint(equalTo: preferredExpandedContentLayoutGuide.heightAnchor)
-        constraint.priority = .defaultHigh // Lower than required so Auto Layout can enforce max sheet height
-        return constraint
-    }()
-
-    private lazy var fullScreenSheetConstraint: NSLayoutConstraint = {
-        let constraint = bottomSheetView.heightAnchor.constraint(equalTo: maxSheetHeightLayoutGuide.heightAnchor)
-        constraint.priority = .defaultHigh // Lower than required so Auto Layout can enforce max sheet height
-        return constraint
-    }()
-
-    private lazy var sheetWidthConstraint: NSLayoutConstraint = bottomSheetView.widthAnchor.constraint(equalTo: view.widthAnchor)
-
-    private lazy var sheetMaxWidthConstraint: NSLayoutConstraint = bottomSheetView.widthAnchor.constraint(lessThanOrEqualToConstant: Constants.maxSheetWidth)
-
-    private lazy var maxSheetHeightLayoutGuide: UILayoutGuide = UILayoutGuide()
+    // Height of the sheet in its most expanded state
+    private var expandedSheetHeight: CGFloat {
+        var height: CGFloat = 0
```

Comment:
I think this can also be a constant, right? (as it's only initialized in the if below and not changed after that)